style.css

```
@@ -0,0 +1,245 @@
+/*************  Main styles  **************/
+
+body {
+  background-color: rgb(236, 238, 241);
+  font-family: "AvenirNext", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+}
+
+li {
+  display: inline-block;
+  margin-left: 20px;
+  margin-right: 20px;
+  padding-top: 30px;
+}
+
+form {
+  background-color: rgb(255, 255, 255);
+  border: 1px solid #c7c7c7;
+  border-radius: 2px;
+  width: 350px;
+  margin: auto;
+  height: 80%;
+  margin-top: 15px;
+}
+
+label {
+  display: block;
+  margin-bottom: 6px;
+}
+
+input {
+  width: 95%;
+  display: block;
+  margin-top: 6px;
+  margin-left: 6px;
+  border: none;
+  outline-offset: 0;
+  outline: none;
+}
+
+button {
+  background-color: #0077c5;
+  border: none;
+  border-radius: 4px;
+  color: white;
+  box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.75);
+}
+
+header {
+  width: 280px;
+  margin-left: auto;
+  line-height: 3px;
+}
+
+footer {
+  width: 40%;
+  margin: auto;
+}
+
+.material-icons {
+  text-shadow: 1px 1px gray;
+}
+
+/***************  Header ******************/
+
+.sign-up > span:nth-child(1) {
+  padding-top: 5px;
+}
+
+#head-link {
+  color: #0077c5;
+}
+
+/***************  Logos *******************/
+
+.main-logo {
+  display: block;
+  width: 118px;
+  height: 35px;
+  margin: auto;
+  margin-bottom: 20px;
+  margin-top: 20px;
```

You could have used one line only.
```suggestion
  margin: 20px auto;
```